### PR TITLE
Reactive changes to switch from ShortcutPath to ExtensionInstallatioRoot

### PR DESCRIPTION
**Pull request by jinujoseph**
@tinaschrepfer @srivatsn @basoundr @dotnet/project-system

### CODE REVIEW

File: src/VisualStudioEditorsSetup/source.extension.vsixmanifest [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <Identity Id="F6B5EACA-7FA1-4591-8D40-A38234763621" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>VisualStudio Editors </DisplayName>
     <Description>Microsoft VisualStudio Editors</Description>
-    <ShortcutPath>..\CommonExtensions\Microsoft\VisualStudio\Editors</ShortcutPath>
+    <ExtensionInstallationRoot>..\CommonExtensions\Microsoft\VisualStudio\Editors</ExtensionInstallationRoot>
```
Comment (tinaschrepfer) [inline src/VisualStudioEditorsSetup/source.extension.vsixmanifest]: This needs to be a project property.

Comment (tinaschrepfer) [inline src/VisualStudioEditorsSetup/source.extension.vsixmanifest]: Also need to separate out the root and folder.

**Comment by davkean** [comment]
In the future, instead of two commits for the same change, you can do the following to end up with the same commit that you just PR into multiple branches:

Find the last common commit between the branches you want to merging into:
```
> git merge-base upstream/Dev15-rc upstream/master
23898b16f8380a67294f272382e11d06b6b45cf0
```

Create a branch based on that commit
```
> git branch MyFix 23898b
```

Make your changes, and this can be merged into both branches without conflicts (or "extra commits").

**Comment by davkean** [comment]
The build issues look related to your change:

```
C:\Users\dotnet-bot\.nuget\packages\Microsoft.VSSDK.BuildTools\15.0.25929-RC2\tools\VSSDK\Microsoft.VsSDK.targets(696,5): error MSB4018: The "EnableExtension" task failed unexpectedly. [D:\j\workspace\windows_debug---65a0f312\src\VisualStudioEditorsSetup\VisualStudioEditorsSetup.csproj]
22:51:14 C:\Users\dotnet-bot\.nuget\packages\Microsoft.VSSDK.BuildTools\15.0.25929-RC2\tools\VSSDK\Microsoft.VsSDK.targets(696,5): error MSB4018: System.NullReferenceException: Object reference not set to an instance of an object. [D:\j\workspace\windows_debug---65a0f312\src\VisualStudioEditorsSetup\VisualStudioEditorsSetup.csproj]
22:51:14 C:\Users\dotnet-bot\.nuget\packages\Microsoft.VSSDK.BuildTools\15.0.25929-RC2\tools\VSSDK\Microsoft.VsSDK.targets(696,5): error MSB4018:    at Microsoft.VsSDK.Build.Tasks.ExtensionManagerUtilities.get_ConfigurationChangedReasonKey() [D:\j\workspace\windows_debug---65a0f312\src\VisualStudioEditorsSetup\VisualStudioEditorsSetup.csproj]
22:51:14 C:\Users\dotnet-bot\.nuget\packages\Microsoft.VSSDK.BuildTools\15.0.25929-RC2\tools\VSSDK\Microsoft.VsSDK.targets(696,5): error MSB4018:    at Microsoft.VsSDK.Build.Tasks.ExtensionManagerUtilities.UpdateLastExtensionsChange(SettingsManager settingsManager) [D:\j\workspace\windows_debug---65a0f312\src\VisualStudioEditorsSetup\VisualStudioEditorsSetup.csproj]
22:51:14 C:\Users\dotnet-bot\.nuget\packages\Microsoft.VSSDK.BuildTools\15.0.25929-RC2\tools\VSSDK\Microsoft.VsSDK.targets(696,5): error MSB4018:    at Microsoft.VsSDK.Build.Tasks.EnableExtension.Execute() [D:\j\workspace\windows_debug---65a0f312\src\VisualStudioEditorsSetup\VisualStudioEditorsSetup.csproj]
22:51:14 C:\Users\dotnet-bot\.nuget\packages\Microsoft.VSSDK.BuildTools\15.0.25929-RC2\tools\VSSDK\Microsoft.VsSDK.targets(696,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() [D:\j\workspace\windows_debug---65a0f312\src\VisualStudioEditorsSetup\VisualStudioEditorsSetup.csproj]
22:51:14 C:\Users\dotnet-bot\.nuget\packages\Microsoft.VSSDK.BuildTools\15.0.25929-RC2\tools\VSSDK\Microsoft.VsSDK.targets(696,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext() [D:\j\workspace\windows_debug---65a0f312\src\VisualStudioEditorsSetup\VisualStudioEditorsSetup.csproj]
22:51:17   DeployTestDependencies -> D:\j\workspace\windows_debug---65a0f312\bin\Debug\Tests\DeployTestDependencies.dll
22:51:22   Microsoft.VisualStudio.ProjectSystem.Managed.TestServices.UnitTests -> D:\j\workspace\windows_debug--
```

Remember that the build machines are running older versions of VS.

**Pull request closed without merging**